java/src/main/java/com/saucelabs/simplesauce/SauceDriverImpl.java

```diff
@@ -9,10 +9,8 @@
 public class SauceDriverImpl implements SauceRemoteDriver {
     private RemoteWebDriver remoteDriver;
 
-    public RemoteWebDriver createRemoteWebDriver(String seleniumServer, MutableCapabilities capabilities)
-        throws MalformedURLException
-    {
-        remoteDriver = new RemoteWebDriver(new URL(seleniumServer), capabilities);
+    public RemoteWebDriver createRemoteWebDriver(URL seleniumServer, MutableCapabilities capabilities) {
```

Comment:
This will cause problems down below where @mmerrell and myself have concerns. I'd like to make sure that we're catching this exception and not forcing users to do `public void test() throws MalformedURLException ` for every test.